### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by christoc-ibm**
Added Cloud Foundry deprecation notice

### CODE REVIEW

**Review by nfstein — CHANGES_REQUESTED** [review]
We dont deploy vapor to CF, please remove any references to CF we still have in there

**Comment by christoc-ibm** [comment]
Removed all CF references.

File: README.md [hunk 1]
```diff
@@ -79,7 +79,6 @@ ibmcloud dev run
 Your application is launched locally. When you are ready to deploy to IBM Cloud on Cloud Foundry or Kubernetes, run one of the following commands:
```
Comment (nfstein) [inline README.md]: one here

Comment (christoc-ibm) [reply README.md]: Dang! Good catch. Removed.

**Review by nfstein — APPROVED** [review]

**Pull request merged**